openmls/src/group/core_group/mod.rs

```diff
@@ -55,10 +55,9 @@ use serde::{
     ser::{SerializeStruct, Serializer},
     Deserialize, Deserializer, Serialize,
 };
-#[cfg(any(feature = "test-utils", test))]
-use std::collections::BTreeMap;
+#[cfg(test)]
```

Comment:
How about we move all these things into a separate module that is guarded by `#[test]` and `impl` blocks guarded by `#[test]`? Then we can declutter all of this.

Same goes for all the other `test` changes in the other files.

Reply:
Fully agree, but I'd prefer to do it as part of a follow-up PR (in the context of #691).